packages/core/src/components/Tooltip/story.tsx

```diff
@@ -214,3 +214,33 @@ export function toggleWithClick() {
 toggleWithClick.story = {
   name: 'Toggle tooltip on click',
 };
+
+export function withAccessibilityLabel() {
+  return (
+    <>
+      <Spacing bottom={1}>
+        <Tooltip
+          accessibilityLabel="Override accessibility content"
+          content="I have an aria-labelledby instead of aria-label"
```

Comment:
noob q: is this the other way arround? 

(i.e here because we added accessibelityLabel, this _has_ an aria-label instead of an aria-labelledby)

Reply:
yes! good catch 😅 I swapped these around a couple times and tho I got the `children` text right I failed to swap the `content` 🙏